Content.Server/Damage/Systems/ExaminableDamageSystem.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Linq;
+using Content.Server.Damage.Components;
+using Content.Server.Destructible;
+using Content.Server.Destructible.Thresholds.Triggers;
+using Content.Shared.Damage;
+using Content.Shared.Damage.Prototypes;
+using Content.Shared.Examine;
+using Content.Shared.Rounding;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Damage.Systems;
+
+public class ExaminableDamageSystem : EntitySystem
+{
+    [Dependency] private readonly IPrototypeManager _prototype = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<ExaminableDamageComponent, ComponentInit>(OnInit);
+        SubscribeLocalEvent<ExaminableDamageComponent, ExaminedEvent>(OnExamine);
+    }
+
+    private void OnInit(EntityUid uid, ExaminableDamageComponent component, ComponentInit args)
+    {
+        if (component.MessagesProtoId == null)
+            return;
+        component.MessagesProto = _prototype.Index<ExaminableDamagePrototype>(component.MessagesProtoId);
+    }
+
+    private void OnExamine(EntityUid uid, ExaminableDamageComponent component, ExaminedEvent args)
+    {
+        if (component.MessagesProto == null)
+            return;
+
+        var messages = component.MessagesProto.Messages;
+        if (messages.Length == 0)
+            return;
+
+        var level = GetDamageLevel(uid, component);
+        var msg = Loc.GetString(messages[level]);
+        args.PushMarkup(msg);
+    }
```

Comment:
If damage is predicted could this be on the client instead? Just saves the bandwidth.

Reply:
Well, damageable is predicted, but `DestructibleComponent` isn't...